### DISCUSSION TITLE
Adjustable stereo level slope setting

### DISCUSSION THREAD

**Discussion by AmateurAudioDude** [PE5PVB/TEF6686_ESP32#634] [Ideas]
An observation, the `Set_Stereo_Level` default slope setting for the TEF module is 24 dB, whereas code for the TEF radio is set to 6 dB. It could be useful to have an option in the Audio settings menu ranging from 6-30dB, particularly useful for reducing stereo noise on webservers.

**Comment by PE5PVB** [comment]
The current setting is finetuned to have the best behaviour. You can tweak this a bit with the signal offset option in the menu.
